packages/xstate-store/src/types.ts

```diff
@@ -385,7 +394,9 @@ export type AnyAtom = Atom<any>;
  * atom.set(43);
  * ```
  */
-export interface ReadonlyAtom<T> extends Readable<T> {}
+export interface ReadonlyAtom<T> extends BaseAtom<T>, Dependency, Subscriber {
+  update(): boolean;
```

Comment:
should `update` be on the atom? I thought it would be exclusive to computeds

Reply:
It is exclusive to computeds (`ReadonlyAtom` is the computed atom)